grafana/download.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/sh
 
-if [ "$1" = "raspberry-pi" ]; then
-	wget -O /tmp/grafana.deb https://dl.grafana.com/oss/release/grafana-rpi_6.2.5_armhf.deb;
+if [ "$1" = "rpi" ]; then
+	wget -O /tmp/grafana.deb https://dl.grafana.com/oss/release/grafana-rpi_6.3.6_armhf.deb;
```

Comment:
Just shell scripting, but the both `/tmp/grafana.deb` and say `https://dl.grafana.com/oss/release/` could be variables, that would result in fewer lines.
E.g.

```
outfile="/tmp/grafana.deb"
download_base="https://dl.grafana.com/oss/release/"
case $1 in
   rpi)  package_file="grafana-rpi_6.3.6_armhf.deb"
       ;;
   aarch61) package_file="grafana_6.3.6_arm64.deb"
       ;;
    *) package_file="grafana_6.3.6_armhf.deb"
esac
wget -O "${outfile}" "${download_base}${package_file}"
```
or something similar. Less duplication, fewer chances for typos.